Added existing website content to sawtooth-website repo

vaporos opened the pull request.

vaporos commented:
When using vagrant, these steps should be automatically performed during the first 'vagrant up' command.  Are you using vagrant?

If you aren't using vagrant, then these are very close to the right steps.  Inside vagrant though, we generally don't do an install and instead have PYTHONPATH setup.  It would be interesting to find an elegant way to allow either vagrant or install into a clean system (Ubuntu and CentOS would be nice, IMO) prior to the tutorial (choose your own adventure), if we can keep the overall complexity down.

jeremyeder commented:
That surprised me too.  I was using your Vagrantfile.  I see now the bootstrap.d/20...file that should have done it.  Hmm.

jeremyeder commented:
As far as elegant way ... the fabric project has a packer-based build pipeline that generates container images.  Something like that would be ideal (container images).

vaporos commented:
Can you provide the output of 'vagrant box outdated'?  I'm on 20160406.0.0; some very old versions of this box apparently don't configure the repo we need for swig3.

jeremyeder commented:
```
$ vagrant box outdated
Checking if box 'ubuntu/trusty64' is up to date...
$

$ vagrant box list | grep ubuntu
ubuntu/trusty64              (virtualbox, 20160406.0.0)
ubuntu/trusty64              (virtualbox, 20160509.1.0)
```

I had run vagrant box update earlier today which grabbed the new one.

christo4ferris commented:
@vaporos bump

vaporos commented:
Since this seems like an issue with outdated vagrant boxes, not a documentation problem per-se, resolving this PR.  Welcome some method to check for new-enough vagrant boxes in the sawtooth-dev-tools repo (maybe 'vagrant up' should fail early).

vaporos committed:
Add empty top-level index.html

We have an empty top-level index.html to ensure that we do not
accidentally list the index of the site. Currently, this page
is never seen since we use a redirect at the top level to /docs/.

Signed-off-by: Shawn T. Amundson <amundson@bitwise.io>

vaporos committed:
Add example website content

This example content currently exists on the website and was
contributed by Intel from its older 01.org Sawtooth Lake
demo website.

Signed-off-by: Shawn T. Amundson <amundson@bitwise.io>

vaporos committed:
Add existing docs/ directory

This includes all of the files which currently make up the
docs/ part of the website, minus the actual docs themselves.

The version.json files are used by the sphinx docs to populate
the version pulldown.

Signed-off-by: Shawn T. Amundson <amundson@bitwise.io>

vaporos committed:
Add inital MAINTAINERS.md

An initial list of those that have been active on the website, docs,
etc.

Signed-off-by: Shawn T. Amundson <amundson@bitwise.io>

ineffectualproperty commented on MAINTAINERS.md:
```diff
@@ -0,0 +1,13 @@
+| Dan Middleton | dcmiddle | Dan |
+| Darian Plumb | dplumb94 | dplumb |
+| James Mitchell | jsmitchell | jsmitchell |
+| Kelly Olson | kelly\_ | ineffectualproperty |
```
Looks like Github and RocketChat got switched. Github is ineffectualproperty, RocketChat is kelly_

vaporos replied:
Fixed.
